### ISSUE TITLE
build-using-dockerfile doesn't import the source image's configuration correctly

### ISSUE THREAD

**Issue by nalind** [projectatomic/buildah#97]
@TomSweeneyRedHat noticed that `build-using-dockerfile` doesn't correctly import the configuration from the source image (the `RunConfig` that we pass in to the imagebuilder Builder is mostly empty), and is also lossy at saving some of the configuration settings which are set in the Dockerfile (we don't import everything from the Builder's ContainerConfig).  Fixing this will overlap a bit with #85.

**Comment by nalind** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [[buildah] Fix build-using-dockerfile's handling of image configurations](https://trello.com/c/nYY2wUAF/478-buildah-fix-build-using-dockerfile-s-handling-of-image-configurations)

**Comment by nalind** [comment]
@TomSweeneyRedHat, if you've got that Dockerfile that demonstrated the bug for you, can you attach it here?  It (and any others) will make useful additions to the tests as part of the PR to fix this.  Thanks!

**Comment by TomSweeneyRedHat** [comment]
Thanks for the ping @nalind.  I've attempted to attach the Dockerfiles here on this response.  If that fails I created a repo on GitHub with them and a couple others at:  https://github.com/TomSweeneyRedHat/TomsDockerfiles.  The files of note are:  Dockerfile.HelloFromContainer, HelloFromContainer.py and you might also try Dockerfile.LsRoot.  The first ones display a "Hello World" type of message 10 times, the second does a list of root after loading up Fedora.  Both hiccup under buildah and work fine with docker.

[Dockerfile.HelloFromContainer.txt](https://github.com/projectatomic/buildah/files/988279/Dockerfile.HelloFromContainer.txt)
[Dockerfile.LsRoot.txt](https://github.com/projectatomic/buildah/files/988281/Dockerfile.LsRoot.txt)
[HelloFromContainer.py.txt](https://github.com/projectatomic/buildah/files/988280/HelloFromContainer.py.txt)
